source/agora/consensus/data/genesis/Coinnet.d

```diff
@@ -183,39 +183,40 @@ unittest
     testSymmetry(GenesisBlock);
 }
 
+// These are used to in package.d to generate the enrollments for replacement if the keys change
+public immutable KeyPair[] genesis_frozen_output_keys = [
+    NODE2,
+    NODE3,
+    NODE4,
+    NODE5,
+    NODE6,
+    NODE7
+    ];
+
 // TODO: Replace with the node's pubkey
 /// NODE2: GCN2C4X4ZAUTENP3VYVA3225CUNEAHDKFXDETNRBXZCXUPIJUY64UANT
-public immutable PublicKey NODE2_ADDRESS =
-    GCN2C4X4ZAUTENP3VYVA3225CUNEAHDKFXDETNRBXZCXUPIJUY64UANT.address;
+public immutable KeyPair NODE2 =
+    GCN2C4X4ZAUTENP3VYVA3225CUNEAHDKFXDETNRBXZCXUPIJUY64UANT;
```

Comment:
No it shouldn't be a `KeyPair`, because we will only have the addresses accessible in the near future, unlike for `Test`.

Reply:
🤦  Will revert.